distribution/ddf-common/src/main/resources/NOTICE

```diff
@@ -110,6 +110,9 @@ This product includes software developed by
 JJ2000 partners
 Licensed under the LGPL and JJ2000 license.
 
+This software is based in part on the work of the
+Independent JPEG Group.
```

Comment:
This entry was added for a [specific file](https://github.com/FFmpeg/FFmpeg/blob/master/libavcodec/jfdctfst.c) used by FFmpeg which is mentioned in the [licensing guide](https://github.com/FFmpeg/FFmpeg/blob/master/LICENSE.md) (it's a special requirement for this part of the code in particular). Should there be a separate entry for FFmpeg itself?